### PR TITLE
SC-2769 - clickable-tags-searchbar-improvement

### PR DESCRIPTION
**Pull request by ABialoskorski**
# Description

## Links to Tickets or other pull requests

- https://ticketsystem.schul-cloud.org/browse/SC-2769

## Changes
#tags are clickable. provider names, too, changed data formatting, enhanced searchbar functionality, now it search after user input, dont wait for him to click search, when the query is already in link it searches for it on page load, before nothing happened

## Screenshots of UI changes
image

### CODE REVIEW

**Comment by ABialoskorski** [comment]
@schul-cloud-bot update screenshots

File: src/components/molecules/ContentSearchbar.vue [hunk 1]
```diff
@@ -132,11 +143,11 @@ export default {
 			align-items: center;
 			justify-content: center;
 			height: 100%;
-			font-size: var(--heading-2);
+			font-size: var(--heading-6);
```
Comment (luizzaly) [inline src/components/molecules/ContentSearchbar.vue:146]: seems bit small, what about --heading-4 on mobile?

File: src/components/molecules/EdusharingFooter.vue [hunk 2]
```diff
@@ -14,12 +14,12 @@
 <style lang="scss">
 .edusharing-footer {
 	display: flex;
-	align-items: flex-end;
 	justify-content: center;
 	width: 100%;
 	margin-top: var(--space-xl);
```
Comment (luizzaly) [inline src/components/molecules/EdusharingFooter.vue:19]: Do we need the margin-top? without it seems to be better visible on mobile devices  :)

File: src/components/molecules/ContentSearchbar.vue (hunk 1 above)
Comment (ABialoskorski) [reply src/components/molecules/ContentSearchbar.vue:146]: done ;)

File: src/components/molecules/EdusharingFooter.vue (hunk 2 above)
Comment (ABialoskorski) [reply src/components/molecules/EdusharingFooter.vue:19]: it looks like requested by the UX, I will leave it as it is

File: locale/de.json [hunk 3]
```diff
@@ -220,10 +220,11 @@
 	"pages.content._id.metadata.createdAt": "erstellt am",
 	"pages.content._id.metadata.provider": "Herausgeber",
 	"pages.content._id.metadata.updatedAt": "zuletzt geändert am",
+	"pages.content._id.metadata.noTags": "Keine Tags",
 	"pages.content.empty_state.error.message": "Achte darauf, dass alle Wörter richtig geschrieben sind. <br>Probiere es mit anderen Suchbegriffen.<br>Probiere es mit allgemeineren Suchbegriffen.<br>Probiere es mit weniger Suchbegriffen.<br>",
 	"pages.content.empty_state.error.oops": "Oops",
 	"pages.content.empty_state.error.subtitle": "Vorschlag:",
-	"pages.content.empty_state.error.title": "<span>Ooopsi, keine Ergebnisse!</span>",
```
Comment (mwolowyk-cap) [inline locale/de.json:226]: Was in UX requirements.

File: src/components/molecules/ContentSearchbar.vue [hunk 4]
```diff
@@ -78,7 +85,7 @@ export default {
 			this.$refs.searchInput.focus();
 		},
 		backspaceKeyHandler(e) {
-			if (!this.isActive && (e.keyCode === 8 || e.keyCode === 27)) {
+			if (e.keyCode === 8 || e.keyCode === 27) {
 				this.$refs.searchInput.focus();
 				this.inputValue = "";
```
Comment (mwolowyk-cap) [inline src/components/molecules/ContentSearchbar.vue:90]: if I press the backspace the whole search string is removed. What if I want to fix the typo?

**Pull request merged**